code/src/java/pcgen/system/ConfigurationSettings.java

```diff
@@ -93,19 +85,6 @@ public static void setCountry(String country)
 		setSystemProperty(USER_COUNTRY, country);
 	}
 
-	/**
-	 * @return the current user directory
-	 */
-	public static String getUserDir()
-	{
-		return SystemUtils.USER_DIR;
-	}
-
-	public static ConfigurationSettings getInstance()
```

Comment:
I don't see the value of eliminating this and forcing many more nulls into other places in the code.  I think localizing the use of null is better